### PR TITLE
Added ruleset for spreadprivacy.com

### PR DESCRIPTION
**Pull request by Jochen-A-Fuerbacher**

### CODE REVIEW

File: src/chrome/content/rules/SpreadPrivacy.com.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!--
+	No alternative certificate subject name matches target host name:
+		www.spreadprivacy.com
```
Comment (YegorIevlev) [inline src/chrome/content/rules/SpreadPrivacy.com.xml]: Mismatched, CN=spread-privacy.ghst.pro.

File: src/chrome/content/rules/SpreadPrivacy.com.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!--
+	Mismatched, CN=spread-privacy.ghst.pro
+		www.spreadprivacy.com
+-->
```
Comment (YegorIevlev) [inline src/chrome/content/rules/SpreadPrivacy.com.xml]:
```xml
<!--
	Mismatched:
		- www (CN=spread-privacy.ghst.pro)
-->
```

File: src/chrome/content/rules/SpreadPrivacy.com.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!--
```
Comment (J0WI) [inline src/chrome/content/rules/SpreadPrivacy.com.xml]: Please add the relation to DDG.

**Comment by J0WI** [comment]
`DuckDuckGo.xml` needs to be updated too (same for #15579).

**Comment by J0WI** [comment]
@Jochen-A-Fuerbacher please have a look at the failing fetch test.

**Pull request merged**